pymatviz/histogram.py

```diff
@@ -74,7 +74,7 @@ def elements_hist(
     """
     ax = ax or plt.gca()
 
-    elem_counts = count_elements(formulas, count_mode)
+    elem_counts = count_elements(formulas, count_mode, fill_value=0)
     non_zero = elem_counts[elem_counts > 0].sort_values(ascending=False)
```

Comment:
tag myself: If pandas series could automatically filter out None, this could be reverted.

Reply:
not sure if this is still relevant but isn't that what `pd.Series.dropna()` is for?

Reply:
Yep I forgot to resolve this comment because I wasn't too sure about the behaviour of `elem_counts > 0`, and it turns out it would filter out None.